client.go

```diff
@@ -18,23 +18,41 @@ package dgo
 
 import (
 	"context"
+	"crypto/x509"
 	"fmt"
+	"log"
 	"math/rand"
+	"net/url"
 	"strings"
 	"sync"
 
 	"github.com/dgraph-io/dgo/v200/protos/api"
+	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials"
 	"google.golang.org/grpc/metadata"
 	"google.golang.org/grpc/status"
 )
 
+var slashPort = "443"
+
 // Dgraph is a transaction aware client to a set of Dgraph server instances.
 type Dgraph struct {
 	jwtMutex sync.RWMutex
 	jwt      api.Jwt
 	dc       []api.DgraphClient
 }
+type authorizationCredentials struct {
+	token string
+}
+
+func (a *authorizationCredentials) GetRequestMetadata(ctx context.Context, uri ...string) (map[string]string, error) {
```

Comment:
Need example and unit test on how this is used.

Reply:
Can he just link to the appropriate interface that is being implemented?

Reply:
What is required here? I have resolved the other requirements.